inspect_data.ipynb

```diff
@@ -39,11 +39,11 @@
     "import matplotlib.lines as lines\n",
     "from matplotlib.patches import Polygon\n",
     "\n",
-    "import utils\n",
-    "import visualize\n",
-    "from visualize import display_images\n",
-    "import model as modellib\n",
-    "from model import log\n",
+    "from mask_rcnn import utils\n",
+    "from mask_rcnn import visualize\n",
+    "from mask_rcnn from visualize import display_images\n",
```

Comment:
from mask_rcnn from visualize construct seems odd?

Reply:
yes, but it was there before... :) OK, I will correct it